xapian-applications/omega/index_file.cc

```diff
@@ -910,12 +929,14 @@ index_mimetype(const string & file, const string & urlterm, const string & url,
 	    sample = p.sample;
 	    author = p.author;
 	    created = p.created;
-	    md5_string(text, md5);
+	    if (md5 == "")
```

Comment:
I'd use `md5.empty()` to test for an empty string in preference to this.

Reply:
I am going to change it.